src/startup_api.py

```
@@ -0,0 +1,338 @@
+            max_length=150,
+            min_length=30,
+            length_penalty=2.0,
+            num_beams=4,
+            early_stopping=True
+        )
+        summary = summarization_model["tokenizer"].decode(outputs[0], skip_special_tokens=True)
+
+        return {
+            "original_text": text,
+            "summary": summary
+        }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)